packages/prosemirror-lwdita-backend/src/server.ts

```diff
@@ -45,5 +45,7 @@ app.get('/', (_req, res) => {
 });
 
 app.listen(3000, () => {
-  console.log('Server is running on ' + config.server.apiUrl);
+  const serverPort = app.settings.port;
+  const serverUrl = app.settings.hostname + ":" + (serverPort == 80 || serverPort == 443) ? "" : ":" + serverPort;
```

Comment:
This won't be necessary at all as the app will always be running on port 3000
as instructed here `app.listen(3000, () => {`

Reply:
Yes at the moment. In future though I thought I could add that to the config

Reply:
@adamretter The node server will always be ran on port 3000, then we use a reverse proxy to expose it.